bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/energy.xml

```diff
@@ -61,7 +91,26 @@
 				<label>Planning</label>
 				<description>Heat planning currently used</description>
 			</channel>
+			<channel id="mode" typeId="energy-mode">
+				<label>Mode</label>
+				<description>Current mode (eg. frost-guard or away)</description>
+			</channel>
+			<channel id="end" typeId="timestamp">
+				<label>Mode End</label>
+				<description>Timestamp until current mode is active</description>
+			</channel>
 		</channels>
 	</channel-group-type>
 
+	<channel-group-type id="energy-battery">
```

Comment:
This already exists

Reply:
Sure ? I did not understand the logic how you create groups and channels fully, but reusing the battery group created useless channels like "battery-level" which is not set by the API, just a textual representation of status

Reply:
I can't test as I do not have this kind of module, so keep it for now, or maybe you can send me an api response for this module.